### PR TITLE
Makes highlighted code easier to read.

### PR DESCRIPTION
**Pull request by saurabh-mish**
Updated the background color which makes the highlighted text easier to read.

### CODE REVIEW

File: layouts/partials/paige/dark.css [hunk 1]
```diff
@@ -6,7 +6,7 @@
 /* LineLink */ .chroma .lnlinks { outline: none; text-decoration: none; color: inherit }
 /* LineTableTD */ .chroma .lntd { vertical-align: top; padding: 0; margin: 0; border: 0; }
 /* LineTable */ .chroma .lntable { border-spacing: 0; padding: 0; margin: 0; border: 0; }
-/* LineHighlight */ .chroma .hl { background-color: #ffffcc }
+/* LineHighlight */ .chroma .hl { background-color: #ffffcc1f }
```
Comment (willfaught) [inline layouts/partials/paige/dark.css:9]: From what I understand, this isn't a valid hex color. Can you link to an example what color this is?

**Review by willfaught — CHANGES_REQUESTED** [review]
Can you explain in more detail what the issue is? Are you referring to the yellow background? What does this change intend to do? Make it a darker yellow?

File: layouts/partials/paige/dark.css (hunk 1 above)
Comment (saurabh-mish) [reply layouts/partials/paige/dark.css:9]: I adjusted the slider under Firefox's developer tools.

Comment (willfaught) [reply layouts/partials/paige/dark.css:9]: It's not a valid color: https://www.color-hex.com/color/ffffcc1f

**Review by saurabh-mish — COMMENTED** [review]
updated color which has a valid hex code

File: layouts/partials/paige/dark.css (hunk 1 above)
Comment (saurabh-mish) [reply layouts/partials/paige/dark.css]: I updated the color - it has a valid hex code and is easy on the eyes

**Comment by willfaught** [comment]
Got it. Looks nicer to me too.

Please squash your commits together into one commit with a title that uses sentence case and the imperative mood without ending punctuation, e.g. "Darken highlighted code for easier reading".

**Comment by saurabh-mish** [comment]
My git-fu is weak

**Comment by willfaught** [comment]
Run:

```
$ git reset --hard ec2b255d4574b7acc5c29b6cd394e80bcd19fd57
$ git rebase --interactive master
```

In the editor, change

```
pick 5a29391 makes highlighted code easier to read.    
pick ec2b255 updated color to valid hex representation  
```

to

```
reword 5a29391 makes highlighted code easier to read.    
fixup ec2b255 updated color to valid hex representation  
```

then save and quit.

In the next editor, replace `makes highlighted code easier to read.` with `Darken highlighted code for easier reading`, then save and quit.

```
$ git push -u origin patch-1 --force
```

**Comment by saurabh-mish** [comment]
In the editor, change

```
pick 5a29391 makes highlighted code easier to read.    
pick ec2b255 updated color to valid hex representation  
```

to

```
reword 5a29391 makes highlighted code easier to read.    
fixup ec2b255 updated color to valid hex representation  
```

I synced `master` and `patch-1` branches and dont see it. I did:

```
git checkout master
git reset --hard ec2b255d4574b7acc5c29b6cd394e80bcd19fd57
git rebase --interactive master
```

The file looks like:

```
noop

# Rebase ec2b255..ec2b255 onto ec2b255 (1 command)
#
# Commands:
# p, pick <commit> = use commit
# r, reword <commit> = use commit, but edit the commit message
# e, edit <commit> = use commit, but stop for amending
# s, squash <commit> = use commit, but meld into previous commit
# f, fixup [-C | -c] <commit> = like "squash" but keep only the previous
#                    commit's log message, unless -C is used, in which case
#                    keep only this commit's message; -c is same as -C but
#                    opens the editor
# x, exec <command> = run command (the rest of the line) using shell
# b, break = stop here (continue rebase later with 'git rebase --continue')
# d, drop <commit> = remove commit
# l, label <label> = label current HEAD with a name
# t, reset <label> = reset HEAD to a label
# m, merge [-C <commit> | -c <commit>] <label> [# <oneline>]
#         create a merge commit using the original merge commit's
#         message (or the oneline, if no original merge commit was
#         specified); use -c <commit> to reword the commit message
# u, update-ref <ref> = track a placeholder for the <ref> to be updated
#                       to this position in the new commits. The <ref> is
#                       updated at the end of the rebase
#
# These lines can be re-ordered; they are executed from top to bottom.
#
# If you remove a line here THAT COMMIT WILL BE LOST.
#
# However, if you remove everything, the rebase will be aborted.
#
```

**Comment by willfaught** [comment]
>git checkout master
>git reset --hard ec2b255d4574b7acc5c29b6cd394e80bcd19fd57
>git rebase --interactive master

Your branch in your fork is patch-1, not master.

You need to do `git checkout patch-1` before doing the steps I gave.

**Comment by saurabh-mish** [comment]
> then save and quit.

I saved the file using the default name of `git-rebase-todo`.


> In the next editor, replace makes highlighted code easier to read. with Darken highlighted code for easier reading, then save and quit.

I did not see this. Here's what I did in the entirety:

```
git init
git remote add origin git@github.com:saurabh-mish/paige.git
git fetch --all

git checkout origin/patch-1
git reset --hard ec2b255d4574b7acc5c29b6cd394e80bcd19fd57
git rebase --interactive origin/master
```

Made suggested replacements in the file, then it using the default name `git-rebase-todo`.

```
git status               # just showed the above file
git add git-rebase-todo
git commit -m "Darken highlighted code for easier reading"
git branch patch-1 ac3fcec
git push -u origin patch-1 --force
```

---

If this is getting tedious for you, I could create a new PR with the change.

**Comment by saurabh-mish** [comment]
creating PR with a new branch

**Pull request closed without merging**